sdk/storage/azure-storage-blob/azure/storage/blob/_shared/response_handlers.py

```diff
@@ -31,6 +31,22 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+class PartialBatchErrorException(HttpResponseError):
+    """There is a partial failure in batch operations.
+
+    :param deserialize: A deserializer
+    :param response: Server response to be deserialized.
+    """
+
+    def __init__(self, message, response, parts):
+
+        self.response = response
```

Comment:
`self.response` and self.message` are assigned in the super HttpResponseError - so you probably don't need them here.